config/fpga/analog_slice_cfg.yml

```diff
@@ -1,16 +1,16 @@
 dt: 0.1e-6
 func_order: 1
 func_numel: 512
-func_domain: [0.0, 3.19375e-9]
+func_domain: [0.0, 1.0e-9]
 func_widths: [18, 18]
 func_exps: [-16, -16]
 chunk_width: 8
 num_chunks: 4
 slices_per_bank: 4
 num_banks: 4
 pi_ctl_width: 9
-vref_rx: 0.4
-vref_tx: 0.4
+vref_rx: 0.3
```

Comment:
In the simulation baseline, voltages vary from 0.1 to 0.4V, hence the change.  This was particularly an issue for ADC input-referred noise, because larger signals are less impacted by the same noise addition